R30.s

```diff
@@ -259,7 +259,7 @@ DOTICK		CAF	1SEC		# RE-REQUEST TICKTEST.
 		MASK	V82FLAGS
 		INDEX	A
 		TC	+1
-		TC	TASKOVER	# IF NO FLAGBITS SET DONT' CHANGE TFF OR
+		TC	TASKOVER	# IF NO FLAGBITS SET DON'T CHANGE TFF OR
```

Comment:
Both are incorrect - apostrophe shouldn't be there. [Source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0518.jpg)